Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -21,6 +21,7 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
+		077ACA6D1CBD6B7200B3A708 /* UIKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = UIKit.framework; path = Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS9.3.sdk/System/Library/Frameworks/UIKit.framework; sourceTree = DEVELOPER_DIR; };
```

Comment:
Commit 0711a87 corrects the path. 
This was caused by the SDK defaulting to the Mac OS SDK. Commit 5fc7ee0 sets the SDK to iOS